contracts/FantomAuction.sol

```diff
@@ -608,13 +598,6 @@ contract FantomAuction is
         // Check auction not already resulted
         require(!auction.resulted, "auction already resulted");
 
-        // Gets info on auction and ensures highest bid is less than the reserve price
-        HighestBid storage highestBid = highestBids[_nftAddress][_tokenId];
-        require(
-            highestBid.bid < auction.reservePrice,
-            "Highest bid is currently above reserve price"
-        );
```

Comment:
This is not what we should allow to do - it will allow the seller (and only the seller) to cancel successful auction. This is not what we want.

To replace the `resultFailedAuction`, I recommend to change `withdrawBid()` function to skip `_endTime` check when bid < reservePrice - it will do what we have discussed and will not change the api significantly.

Reply:
But doesn't that mean we allow immediate withdrawals?